.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 
 before_install:
-  - npm install -g npm@latest
+  - npm install -g npm@5
```

Comment:
The latest version of `npm` does not work on the earlier versions of Node, hence why I pinned it